restapi/src/main/java/io/stargate/web/impl/Server.java

```diff
@@ -104,7 +116,21 @@ protected void configure() {
     environment.jersey().register(TablesResource.class);
     environment.jersey().register(KeyspacesResource.class);
     environment.jersey().register(ColumnsResource.class);
-    environment.jersey().register(SwaggerStaticResource.class);
+
+    environment.jersey().register(ApiListingResource.class);
```

Comment:
Does this still return raw json/yaml with `/swagger.json` and `/swagger.yaml`?

Reply:
```

Reply:
Sweet